PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -342,6 +360,10 @@ GradientDistortionCoeffs=`opts_GetOpt1 "--gdcoeffs" $@`
 AvgrdcSTRING=`opts_GetOpt1 "--avgrdcmethod" $@`
 TopupConfig=`opts_GetOpt1 "--topupconfig" $@`
 BiasFieldSmoothingSigma=`opts_GetOpt1 "--bfsigma" $@`
+CustomBrain=`opts_GetOpt1 "--custombrain" $@`
+
+# Defaults
+CustomBrain=`opts_DefaultOpt $CustomBrain NONE`
```

Comment:
Move down several lines, where we also set the default on `UseJacobian`.

Reply:
i.e., I would reorder this section as follows:
```
CustomBrain=`opts_GetOpt1 "--custombrain" $@`

#NOTE: currently is only used in gradient distortion correction of spin echo fieldmaps to topup
#not currently in usage, either, because of this very limited use
UseJacobian=`opts_GetOpt1 "--usejacobian" $@`
# Convert UseJacobian value to all lowercase (to allow the user the flexibility to use True, true, TRUE, False, False, false, etc.)
UseJacobian="$(echo ${UseJacobian} | tr '[:upper:]' '[:lower:]')"

# Use --printcom=echo for just printing everything and not actually
# running the commands (the default is to actually run the commands)
RUN=`opts_GetOpt1 "--printcom" $@`

# Defaults
UseJacobian=`opts_DefaultOpt $UseJacobian "true"`
CustomBrain=`opts_DefaultOpt $CustomBrain NONE`
```

Reply:
Done.